larky-api/src/main/java/com/verygood/security/larkyapi/LarkyAPIEngine.java

```diff
@@ -0,0 +1,39 @@
+package com.verygood.security.larkyapi;
+
+import com.verygood.security.larky.jsr223.LarkyScriptEngine;
+import com.verygood.security.larky.parser.ParsedStarFile;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptException;
+import java.io.Reader;
+
+public class LarkyAPIEngine extends LarkyScriptEngine {
```

Comment:
Does this have to be called `Engine`? Wouldn't this just be a `LarkyRuntime` or something?